export_lda_viz.py

```diff
@@ -0,0 +1,8 @@
+from bagel.viz.viz import export_lda_viz
+
+FILE_PATTERN = 'zero-carbon-bill/input/*.txt'
+# Tweak this parameter until it brings joy
+NUMBER_OF_TOPICS = 5
+OUTPUT_PATH = 'docs/index_lda.html'
+
+export_lda_viz(NUMBER_OF_TOPICS, FILE_PATTERN, OUTPUT_PATH)
```

Comment:
should be wrapped in main

Reply:
Hahahaha, thank you Brenda, I was thinking about sneakily adding you to review <3

Reply:
does it look all right to merge @Br3nda ?